examples/workflow/qe_scf_calculation.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            "cluster": {"fqdn": "master-production-20160630-cluster-001.exabyte.io"},
</pre><p>Do we need to provide the cluster FQDN?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
It is optional/default, removed.